dusk-labyrinth.lic

```diff
@@ -8,15 +8,15 @@ class DuskLab
   include DRC
   include DRCI
   def initialize
-    arg_definitions = [
-      [{ name: 'loot_bag', display: 'container noun', regex: /\w+/i, description: 'Bag for your loot!' },
-       { name: 'redeem_scrip', display: 'redeem', regex: /^redeem/i, optional: true, description: 'Redeem scrip when finished?' }]
-    ]
-    args = parse_args(arg_definitions)
+    @settings = get_settings
```

Comment:
Does this need to be `settings` (not `@`) now? See the next line `settings...`

Reply:
Ugh, sorry about that. I updated line 12 to be `@settings` per https://github.com/rpherbig/dr-scripts/pull/4390/commits/b3c469630bf8f0c4794f039e6ae5e3af85889b22

Before committing I actually checked that the script would activate without error this time 😬